pkg/ssh/ssh.go

```diff
@@ -50,22 +51,30 @@ func SetAuthorizedKeys(cfg *config.CloudConfig, withNet bool) error {
 }
 
 func getKey(key string, withNet bool) (string, error) {
-	if !strings.HasPrefix(key, "github:") {
+	providers := map[string]string{
+		"github": "https://github.com/%s.keys",
+		"gitlab": "https://gitlab.com/%s.keys",
+	}
+
+	url, err := url.Parse(key)
```

Comment:
Maybe the `url` variable needs another name?

Reply:
i used `uri` in my example because that's usually what I am parsing with this hack

Reply:
I'm just worried that the `url` from the `net/url` package will be overridden if I use a variable with the same name? #go-n00b

Reply:
Just read up a bit on this and I guess we're ok in this specific case as the `url` module isn't used after this. So unless you want me to use a more explicit name for the variable I'll leave this be as it is.